engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -279,40 +280,40 @@ private void initRenderGraph() {
     private void addShadowMapNodes(RenderGraph renderGraph) {
         FBOConfig shadowMapConfig = new FBOConfig(ShadowMapNode.SHADOW_MAP_FBO_URI, FBO.Type.NO_COLOR).useDepthBuffer();
         BufferClearingNode shadowMapClearingNode = new BufferClearingNode(shadowMapConfig, shadowMapResolutionDependentFBOs, GL_DEPTH_BUFFER_BIT);
-        renderGraph.addNode(shadowMapClearingNode, "shadowMapClearingNode");
+        renderGraph.addNode(shadowMapClearingNode, new SimpleUri("engine:shadowMapClearingNode"));
 
         shadowMapNode = new ShadowMapNode(context);
-        renderGraph.addNode(shadowMapNode, "shadowMapNode");
+        renderGraph.addNode(shadowMapNode, new SimpleUri("engine:shadowMapNode"));
     }
 
     private void addReflectionNodes(RenderGraph renderGraph) {
         FBOConfig reflectedBufferConfig = new FBOConfig(BackdropReflectionNode.REFLECTED_FBO_URI, HALF_SCALE, FBO.Type.DEFAULT).useDepthBuffer();
         BufferClearingNode reflectedBufferClearingNode = new BufferClearingNode(reflectedBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
-        renderGraph.addNode(reflectedBufferClearingNode, "reflectedBufferClearingNode");
+        renderGraph.addNode(reflectedBufferClearingNode, new SimpleUri("engine:reflectedBufferClearingNode"));
 
         Node reflectedBackdropNode = new BackdropReflectionNode(context);
-        renderGraph.addNode(reflectedBackdropNode, "reflectedBackdropNode");
+        renderGraph.addNode(reflectedBackdropNode, new SimpleUri("engine:reflectedBackdropNode"));
 
         Node worldReflectionNode = new WorldReflectionNode(context);
-        renderGraph.addNode(worldReflectionNode, "worldReflectionNode");
+        renderGraph.addNode(worldReflectionNode, new SimpleUri("worldReflectionNode"));
     }
 
     private void addSkyNodes(RenderGraph renderGraph) {
         FBOConfig reflectedRefractedBufferConfig = new FBOConfig(RefractiveReflectiveBlocksNode.REFRACTIVE_REFLECTIVE_FBO_URI, FULL_SCALE, FBO.Type.HDR).useNormalBuffer();
         BufferClearingNode reflectedRefractedClearingNode = new BufferClearingNode(reflectedRefractedBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
-        renderGraph.addNode(reflectedRefractedClearingNode, "reflectedRefractedClearingNode");
+        renderGraph.addNode(reflectedRefractedClearingNode, new SimpleUri("engine:reflectedRefractedClearingNode"));
 
-        FBOConfig lastUpdatedGBufferConfig = displayResolutionDependentFBOs.getFboConfig(READONLY_GBUFFER);
-        FBOConfig staleGBufferConfig = displayResolutionDependentFBOs.getFboConfig(WRITEONLY_GBUFFER);
+        FBOConfig gBuffer1Config = displayResolutionDependentFBOs.getFboConfig(new SimpleUri("engine:fbo.gBuffer1")); // TODO: Remove the hard coded value here
```

Comment:
Similarly to an earlier comment, I don't quite understand why the renamings here and below.